### PR TITLE
Use metadata store setModel in addition to setRoot

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
Part of in https://github.com/ome/ome-model/pull/17; see this PR and https://trello.com/c/hnpxmvhe/52-extended-annotation-support-pt-i for description and testing instructions.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
     <testng.version>6.8</testng.version>
     <guava.version>17.0</guava.version>
     <ome-common.version>5.3.1</ome-common.version>
-    <ome-model.version>5.3.1</ome-model.version>
+    <ome-model.version>5.3.2-SNAPSHOT</ome-model.version>
```
Comment (sbesson) [inline pom.xml]: Now ome-model 5.4.0 is out and as per https://github.com/ome/ome-model/commit/cfed4019e445eba49ff7440f967c2e4886be3b98, this should be updated to 6.0.0-SNAPSHOT

Comment (rleigh-codelibre) [inline pom.xml]: Done.

**Comment by sbesson** [comment]
Discussed with @rleigh-codelibre following https://github.com/ome/ome-model/pull/17#issuecomment-271299247, excluding to test the Java stack without this PR.

File: pom.xml [hunk 2]
```diff
@@ -54,7 +54,7 @@
     <testng.version>6.8</testng.version>
     <guava.version>17.0</guava.version>
     <ome-common.version>5.3.1</ome-common.version>
-    <ome-model.version>5.3.1</ome-model.version>
```
Comment (sbesson) [inline pom.xml]: While running the tests with this PR excluded, I realized the ome-model version bump required to couple the model changes to the OMERO `metadata_files` branch was part of it. Would it be possible to move this line into a PR that we can merge separately?

Comment (rleigh-codelibre) [inline pom.xml]: Backed out and moved to https://github.com/openmicroscopy/bioformats/pull/2720

**Comment by sbesson** [comment]
Closing for now. To be revisited when restarting the extended metadata work

**Pull request closed without merging**